synapse/storage/util/sequence.py

```diff
@@ -118,6 +143,18 @@ def check_consistency(
             "SELECT last_value, is_called FROM %(seq)s" % {"seq": self._sequence_name}
         )
         last_value, is_called = txn.fetchone()
+
+        # If we have an associated stream check the stream_positions table.
+        max_in_stream_positions = None
+        if stream_name:
+            txn.execute(
+                "SELECT MAX(stream_id) FROM stream_positions WHERE stream_name = ?",
+                (stream_name,),
+            )
+            row = txn.fetchone()
+            if row and row[0] is not None:
+                max_in_stream_positions = row[0][0]
```

Comment:
Should this now be `row[0]`? (Also the check for `row[0] is not None` isn't really necessary since we'll just set the max back to `None`, but 🤷

Reply:
sigh